ovs-p4rt/ovs_p4rt_session.h

```diff
@@ -28,6 +30,18 @@ inline absl::uint128 TimeBasedElectionId() {
   return absl::MakeUint128(absl::ToUnixSeconds(absl::Now()), 0);
 }
 
+static void readFile(const std::string& filename, std::string& data)
```

Comment:
The static function definition (and the #includes for \<fstream\> and \<sstream\>) are not part of the interface and do _not_ belong in this header file.

You're also not following the coding conventions for this module.